dao/src/main/java/org/thingsboard/server/dao/dashboard/DashboardTitlesRedisCache.java

```diff
@@ -30,6 +30,6 @@
 public class DashboardTitlesRedisCache extends RedisTbTransactionalCache<DashboardId, String> {
 
     public DashboardTitlesRedisCache(TBRedisCacheConfiguration configuration, CacheSpecsMap cacheSpecsMap, RedisConnectionFactory connectionFactory) {
-        super(CacheConstants.DASHBOARD_TITLES_CACHE, cacheSpecsMap, connectionFactory, configuration, new TbJavaRedisSerializer<>());
+        super(CacheConstants.DASHBOARD_TITLES_CACHE, cacheSpecsMap, connectionFactory, configuration, new TbJsonRedisSerializer<>(String.class));
```

Comment:
Maybe some String serializer will be enough? Why we should pass the primitive string through the json with escaping overhead ?